boot/publish.sh

```diff
@@ -3,7 +3,7 @@
 MODS=${MODS:-"lazyload limiter plugin"}
```

Comment:
We can traverse these modules through the file directory so that we don't have to list them explicitly. But this can be done later